test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/TypeEnumeratorTests.cs

```diff
@@ -177,84 +165,81 @@ public void GetTestsShouldNotReturnBaseTestMethodsFromAnotherAssemblyByConfigura
     }
 
-    [TestMethod]
     public void GetTestsShouldNotReturnHiddenTestMethods()
     {
         SetupTestClassAndTestMethods(isValidTestClass: true, isValidTestMethod: true, isMethodFromSameAssembly: true);
         TypeEnumerator typeEnumerator = GetTypeEnumeratorInstance(typeof(DummyHidingTestClass), Assembly.GetExecutingAssembly().FullName);
 
         var tests = typeEnumerator.Enumerate(out _warnings);
 
-        Assert.IsNotNull(tests);
-        Assert.AreEqual(
-            1,
+        Verify(tests is not null);
+        Verify(
+            1 ==
             tests.Count(t => t.TestMethod.Name == "BaseTestMethod"),
             "DummyHidingTestClass declares BaseTestMethod directly so it should always be discovered.");
-        Assert.AreEqual(
-            1,
+        Verify(
+            1 ==
             tests.Count(t => t.TestMethod.Name == "DerivedTestMethod"),
             "DummyHidingTestClass declares BaseTestMethod directly so it should always be discovered.");
-        Assert.IsFalse(
+        Verify(!
             tests.Any(t => t.TestMethod.DeclaringClassFullName == typeof(DummyBaseTestClass).FullName),
             "DummyHidingTestClass hides BaseTestMethod so declaring class should not be the base class");
     }
 
-    [TestMethod]
     public void GetTestsShouldReturnOverriddenTestMethods()
     {
         SetupTestClassAndTestMethods(isValidTestClass: true, isValidTestMethod: true, isMethodFromSameAssembly: true);
         TypeEnumerator typeEnumerator = GetTypeEnumeratorInstance(typeof(DummyOverridingTestClass), Assembly.GetExecutingAssembly().FullName);
 
         var tests = typeEnumerator.Enumerate(out _warnings);
 
-        Assert.IsNotNull(tests);
-        Assert.AreEqual(
-            1,
+        Verify(tests is not null);
+        Verify(
+            1 ==
             tests.Count(t => t.TestMethod.Name == "BaseTestMethod"),
             "DummyOverridingTestClass inherits BaseTestMethod so it should be discovered.");
-        Assert.AreEqual(
-            1,
+        Verify(
+            1 ==
             tests.Count(t => t.TestMethod.Name == "DerivedTestMethod"),
             "DummyOverridingTestClass overrides DerivedTestMethod directly so it should always be discovered.");
-        Assert.AreEqual(
-            typeof(DummyHidingTestClass).FullName,
+        Verify(
+            typeof(DummyHidingTestClass).FullName ==
             tests.Single(t => t.TestMethod.Name == "BaseTestMethod").TestMethod.DeclaringClassFullName,
             "DummyOverridingTestClass inherits BaseTestMethod from DummyHidingTestClass specifically.");
-        Assert.IsNull(
-            tests.Single(t => t.TestMethod.Name == "DerivedTestMethod").TestMethod.DeclaringClassFullName,
+        Verify(
+            tests.Single(t => t.TestMethod.Name == "DerivedTestMethod").TestMethod.DeclaringClassFullName is null,
             "DummyOverridingTestClass overrides DerivedTestMethod so is the declaring class.");
```

Comment:
Please move this arg to be a comment before the `Verify`. It's currently going in the `expression` param which we don't want to be filled by "user".